Sources/PointFree/Episode/Show.swift

```diff
@@ -108,15 +108,23 @@ private let episodeTocView = View<(blocks: [Episode.TranscriptBlock], isEpisodeV
   )
 }
 
-private func timestampLinkAttributes(_ timestamp: Int) -> [Attribute<Element.A>] {
-  return [
-    href("#t\(timestamp)"),
+private func timestampLinkAttributes(timestamp: Int, useAnchors: Bool) -> [Attribute<Element.A>] {
```

Comment:
now we can customize timestamp links based on whether or not we also want to anchor them.